python/beeai_framework/agents/tool_calling/agent.py

```diff
@@ -40,6 +43,8 @@
 
 __all__ = ["ToolCallingAgent"]
 
+from beeai_framework.utils.strings import to_json
```

Comment:
is this import supposed to be below the `__all__`?

Reply:
updated